js/plugins.js

```diff
@@ -27,8 +16,6 @@ module.exports = (ENV) => {
         NODE_ENV: JSON.stringify(ENV || 'development'),
       },
     }),
-    new webpack.NamedModulesPlugin(),
```

Comment:
This is no longer needed. It seems to have been related to a combination of making debugging easier and caching. I haven't noticed any difference since removing it.
For prosterity's sake I'll point out that if we _do_ want this back in the future we can use the following [as per the docs](https://webpack.js.org/migrate/5/#update-outdated-options):
```js
module.exports = {
  //...
  optimization: {
    moduleIds: 'named',
  },
};
```